src/app/edit_config.rs

```diff
@@ -1,12 +1,17 @@
 use std::{collections::HashMap, fmt::Display, path::Path};
 
+use derive_getters::Getters;
+
 use super::config::Config;
 
-#[derive(Debug)]
+#[derive(Debug, Getters)]
 pub struct EditConfigState {
+    #[getter(skip)]
     config_buffer: HashMap<EditableConfig, String>,
+    #[getter(rename = "highlighted")]
     highlighted_entry: usize,
     is_editing: bool,
+    #[getter(rename = "edit")]
```

Comment:
I favor not renaming this. Again, we can be less verbose than `editing_val`, but just `edit` is too ambiguous, e.g.,

```
Span::styled(edit_config_state.edit().to_string(), Style::default()),
```

in `src/ui.rs` won't be clear to an unadvised reader.

Reply:
Maybe `value` would be better?

Reply:
I swear I am not picking on you 🥲, but I repeat the idea that `value` is too ambiguous 😅

What about `curr_val`?

Also, I was thinking about not just renaming the getters, but renaming the struct fields itself. What do you think?